docs/vm-images.md

```diff
@@ -1,61 +1,161 @@
+
 The HPC Toolkit has officially supported the [HPC CentOS 7 VM Image][hpcimage] as the
 primary VM image for HPC workloads on Google Cloud since it's release. Since the
 [HPC CentOS 7 VM Image][hpcimage] comes pre-tuned for optimal performance on
 typical HPC workloads, it is the default VM image in our modules, unless there
 is specific requirement for a different OS distribution.
 
-Exceptions:
-
-* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
-  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
-  github repo. **This requirement only applies to the servers, not the clients.**
-* [omnia-install]: Only provides support for Rocky 8.
-
 [hpcimage]: https://cloud.google.com/blog/topics/hpc/introducing-hpc-vm-images
 
-## Ubuntu
-The HPC Toolkit officially supports Ubuntu based VM images in the majority of
-our modules, with a couple exceptions:
-
-* [htcondor-configure]: Only provides support for the HPC CentOS 7 image.
-* [nfs-server]: Only provides support for CentOS 7 images for the server itself.
-* [DDN-EXAScaler]: The underlying terraform module has a limitted set of
-  supported images [documented][exascalerimages] in the exascaler-cloud-terraform
-  github repo. **This requirement only applies to the servers, not the clients.**
-* [omnia-install]: Only provides support for Rocky 8.
-
-HPC Toolkit modules and startup scripts
-have been evaulated against the `ubuntu-2004-lts` image family. For more
-information about the Ubuntu Google Cloud images, see the Canonical
-[documentation](https://ubuntu.com/server/docs/cloud-images/google-cloud-engine).
-
-To use the Ubuntu images with the `schedmd-slurm-gcp-v5` modules, follow
-the pattern used in the [hpc-slurm-ubuntu2004.yaml] example.
-
-In most other modules that provide the option to set a VM image, you can set it
-to use the Ubuntu image with the following:
-
-```yaml
-...
-settings:
-  instance_image:
-    family: ubuntu-2004-lts
-    project: ubuntu-os-cloud
-```
-
-[htcondor-configure]: ../community/modules/scheduler/htcondor-configure/README.md
-[nfs-server]: ../community/modules/file-system/nfs-server/README.md
-[DDN-EXAScaler]: ../community/modules/file-system/DDN-EXAScaler/README.md
-[exascalerimages]: https://github.com/DDNStorage/exascaler-cloud-terraform/blob/master/gcp/README.md#boot-image-options
-[omnia-install]: ../community/modules/scripts/omnia-install/README.md
-[hpc-slurm-ubuntu2004.yaml]: ../community/examples/hpc-slurm-ubuntu2004.yaml
+## HPC Rocky Linux 8
+
+HPC Rocky Linux 8 is planned to become the primary supported VM image for HPC workloads on Google Cloud from 2024.
+
+## Debian 11
+
+The HPC Toolkit officially supports Debian 11 based VM images in the majority of our modules, with a couple of exceptions.
+
+## Ubuntu 20.04 LTS
+
+The HPC Toolkit officially supports Ubuntu 20.04 LTS based VM images in the majority of
+our modules, with a couple of exceptions.
+
+## Compatibility table
+
+<table>
```

Comment:
nit: How did you generate the html? You might leave a comment in the code with a hit on how to keep up to date in the future. Example:

```
<!---
To update the table you can use http://... to build an html table
-->
```

Reply:
I actually did it by hand.  I used HTML instead of Markdown so I could get the row headers to span multiple rows.  I tried to use CSS to add color to make it more readable, but it doesn't seem to work in GitHub's viewer.